src/docfx/build/redirection/BuildRedirection.cs

```diff
@@ -27,6 +28,11 @@ internal static List<Error> Build(Context context, Document file)
                 file.Mime.Value);
 
             publishItem.RedirectUrl = context.RedirectionProvider.GetRedirectUrl(file.FilePath);
+            var (documentId, _) = context.DocumentProvider.GetDocumentId(context.RedirectionProvider.GetOriginalFile(file.FilePath));
```

Comment:
Do we also want to include `document_version_independent_id`? They seem to appear hand in hand. Though the bug/feature may not say it is required.

Reply:
I can sync with the requester, not required from the description

Reply:
Yes, we do.